talk/owt/sdk/base/sysinfo.cc

```diff
@@ -20,7 +24,40 @@ std::string SysInfo::SdkType() {
 #endif
 }
 #if defined(WEBRTC_WIN)
+using namespace rtc::rtc_win;
 std::string SysInfo::OsName() {
+  Version version = GetVersion();
```

Comment:
I'm removing OS info from ua-info, https://github.com/open-webrtc-toolkit/owt-server/pull/980. For P2P mode, we can use capabilities to know if remote side supports a specific feature.

Reply:
For compatibility I will keep this at present.  You can create follow up PR once server PR #980 is merged.

Reply:
https://github.com/open-webrtc-toolkit/owt-server/pull/980 is merged, I'm going to remove it from all client SDKs. You safely remove it in this PR, or I'll submit another PR to remove it.

Reply:
Let's use a seperate PR for this.